jetstream/argo.py

```diff
@@ -110,14 +123,14 @@ def get_config(
     creds, projects = google.auth.default(scopes=["https://www.googleapis.com/auth/cloud-platform"])
 
     auth_req = google.auth.transport.requests.Request()
-    creds.refresh(auth_req)
+    creds.refresh(auth_req)  # refrsh token
 
     configuration = Configuration()
     configuration.host = f"https://{cluster_ip}"
     with NamedTemporaryFile(delete=False) as ca_cert:
         ca_cert.write(base64.b64decode(cluster_cert))
     configuration.ssl_ca_cert = ca_cert.name
     configuration.api_key_prefix["authorization"] = "Bearer"
-    configuration.api_key["authorization"] = creds.token
+    configuration.api_key["authorization"] = creds.token  # valid for one hour
```

Comment:
This access token is valid [for about an hour](https://cloud.google.com/iam/docs/creating-short-lived-service-account-credentials#sa-credentials-oauth). So we need to refresh it at some point.